ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -74,6 +79,14 @@ const AlertHeader = ({
             </a>
           </Row>
         </Col>
+        {user.isStaff &&
+          alertSummary.original_revision !== alertSummary.revision && (
+            <Col className="p-0" xs="auto">
+              <Button className="ml-1" size="xs" onClick={handleRevertRevision}>
+                Reset hash
```

Comment:
This needs to be changed into `Reset Revision` so it's consistent with the `Edit Revision` button that will come.

Reply:
Done